encrypted-media-respec.html

```diff
@@ -134,13 +134,15 @@
       <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/192">Issue 192</a> - The specification should do more to encourage/ensure CDM-level interoperability.</p>
       <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/221">Issue 221</a> - This specification contains sections for describing <a href="#security">security</a> and <a href="#privacy">privacy</a> considerations. These sections are not final and review is welcome.</p>
 
-      <p>The following features are <strong>at risk</strong> and may be removed due to lack of implementation.
-      </p>
+      <p>The following features are <strong>at risk</strong> and may be removed:
+	  </p>
       <ul>
-        <li><a def-id="persistent-usage-record-session"></a></li>
-        <li><a def-id="media-key-session-destroyed-algorithm"></a></li>
+        <li><p>The <a def-id="persistent-usage-record-session"></a> session type.</p></li>
+        <li><p>The <a def-id="media-key-session-destroyed-algorithm"></a> algorithm.</p></li>
+        <li><p>Setting the <var>media element</var>'s <a def-id="readystate"></a> value based on key availability in the <a def-id="resume-playback-algorithm"></a> algorithm.</p></li>
```

Comment:
We need to cover both "directions":
`<a def-id="queue-waitingforkey-algorithm"></a> and <a def-id="resume-playback-algorithm"></a> algorithms.`